<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="lbl_img">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>931</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ai">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>1221</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>1291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buy1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>740</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>구매하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buy2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>740</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>구매하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_buy3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>740</y>
     <width>221</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>구매하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pd1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pd2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pd3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>410</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pn1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pn2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>660</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pn3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>660</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_more">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>90</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>상품 추천</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
